airgun/entities/subscription.py

```diff
@@ -142,6 +141,15 @@ def pre_navigate(self, _tries, *args, **kwargs):
 
     def step(self, *args, **kwargs):
         self.view.menu.select('Content', 'Subscriptions')
+        wait_for_spinner(self.view)
+        self.parent.browser.plugin.ensure_page_safe()
+        wait_for(
+            lambda: self.am_i_here(*args, **kwargs),
```

Comment:
any other options? i don't get why we aren't on requested page after all the ajax finished and spinner is gone

Reply:
+1

Reply:
The page make some delays to load and yes after all that, without the waiter it work 2 time but fail 3 other times, any way I do see any inconvenient here to put that waiter as we here clicked that button and we already know what we are expecting, with this waiter the things are very stable eg: if we do not do it better to  register a bug about this and put on hold that issue.

Reply:
If there's a bug - better to submit it, but let's not put PR on hold - fixing that bug may take months as it's not critical.